Project 2: Readme-updates (#248)
netlight-johannadagfalk/goclimate: README.md

| `Region`              | A named region with a combination of language and currency. Each region corresponds to their own base URL for SEO reasons. (Not yet implemented.) |
| `User`                | A registered user. We have accounts both with and without subscriptions. Active subscriptions are handled in Stripe, with some metadata duplicated in this model. |
| `CardCharge`          | Charges for purchases of an offset product.         |

> | `ClimateActions`      | High impact, low-effort actions available for users to accept and perform. |

Reviewer: Funderar på om detta är dumt att ta med i vår README-version iom att det kanske blir så att de lanserar våra kodbaser separat? Då kanske inte detta finns när vår kod lanseras

Follow linter config (open a PR with rule changes if you disagree) and write
tests according to [Better Specs](http://www.betterspecs.org).

> For react files this project uses Prettier for stylistic rules and ESlint for code quality rules. To avoid conflicts, ESlint runs Prettier as an ESlint rule. 

Reviewer: ```suggestion
For React files this project uses Prettier for stylistic rules and ESlint for code quality rules. To avoid conflicts, ESlint runs Prettier as an ESlint rule. 
```

disable_experiments=my_feature,my_other_feature
```

##### Specific React flags

> There are two active react experiments for the form and sign up process.

Reviewer: ```suggestion
There are two active React experiments for the form and sign up process.
```

Flaggs that can be enabled and disabled are:

> `V1` & `V2` 

Reviewer: Små bokstäver på dessa överallt så man inte skriver fel när man vill enable/disable

Don't forget the file extension!

##### State handling
This project uses the react hook useContext to handle global states. 

##### Language in React

> Language files (.yml) are reached in react by reaching and loading the file through its path, and package it to a json object. Each specific language code is reached by using I18n.locale[0..-1] that returns e.g. ‘en’. Se example below. The object ‘text’ can then be reached from within the react component by parsing the json object. The object is then made global through useContext.

Reviewer: ```suggestion
Language files (.yml) are reached in React by loading the file through its path as a props, and package it to a json object. Each specific language code is reached by using I18n.locale[0..-1] that returns e.g. ‘en’. Se example below. The received prop ‘text’ can then be reached from within the React component by parsing the json object. The object is then made global through useContext.
```

* PostgreSQL
* Firefox and geckodriver
* For working with localized strings: Crowdin CLI & account (https://support.crowdin.com/cli-tool/)

> * Yarn is the package manager of this code-base.

Reviewer: Känns verkligen som en smaksak men jag nämner det iallafall
```suggestion
* Yarn is the package manager of this codebase.
```

## Developing

* Run the development server: `bin/rails server`

> * If you're doing front-end work, Webpack dev server gives you hot realoding of JS & CSS: `bin/yarn dev`. (Run simultaneously as bin/rails server)

Reviewer: ```suggestion
* If you're doing front-end work, Webpack dev server gives you hot realoding of JS & CSS: `bin/yarn dev`. Run this simultaneously as `bin/rails server`
```

* Watch for updates and continuosly run relevant specs & linters: `bin/guard`
* For changes inside the react-components folder, run eslint and prettier: `bin/yarn lint`

> * Run Ruby tests: `bin/rspec`. (Do not run when webpack dev server is running)

Reviewer: ```suggestion
* Run Ruby tests: `bin/rspec`. Do not run when webpack dev server is running
```
